cognite_toolkit/common_function_code/tool.py

```diff
@@ -129,23 +128,20 @@ def environment_variables(self) -> dict[str, str | None]:
         return {**self._environ.copy(), **os.environ}
 
     def as_string(self) -> str:
-        environment = self._environ.copy()
+        environment = os.environ
```

Comment:
Wouldn't this cause the environment to be changed? If you don't do a `.copy`?

Reply:
Great catch!